releases-beta/how-to-guides/managing-cherry-picks.md

```diff
@@ -0,0 +1,47 @@
+# Managing Cherry-Picks
+
+Cherry-picking is an essential tool in version control systems, allowing you to apply specific commits from one branch to another without merging the entire branch. In Aviator’s release management system, cherry-picks are used to selectively integrate changes into release candidates. This guide will walk you through the process of managing cherry-picks in Aviator, ensuring smooth handling of your release process.
+
+## Prerequisites
+
+Before you begin cherry-picking, ensure that your repository is properly connected to Aviator and that a release project has been created.
+
+1. Set up your repository and connect it to Aviator.
+
+![](../../.gitbook/assets/release-github-integration.png)
+
+2. [Create a release project](./creating-a-release-project.md) if it does not already exist.
+
+## Cherry-Picking a Pull Request (PR)
+
+### Step 1: Navigate to the Aviator Release Dashboard
+
+1. Open Aviator’s Release Dashboard.
+
+2. Cut a release from the listed PRs on the dashboard.
```

Comment:
Mention that this is the release on which you will apply a cherry-pick. So one can skip this step if they already have an existing release to base cherry-pick on.